src/applications/simple-forms/21-0845/pages/limitedInfo.js

```diff
@@ -3,25 +3,36 @@ import React from 'react';
 import { LIMITED_INFORMATION_ITEMS } from '../definitions/constants';
 import GroupCheckboxWidget from '../../shared/components/GroupCheckboxWidget';
 
+const labelString =
+  'Which specific information do you authorize us to release?';
+
 /** @type {PageSchema} */
 export default {
   uiSchema: {
     limitedInformationItems: {
       'ui:title': (
         <>
-          <span className="custom-header vads-u-font-family--serif vads-u-font-weight--bold vads-u-font-size--h3">
-            Which specific information do you authorize us to release?{' '}
-            <span className="custom-required-span hide-on-review-page">
+          <h3>
+            {labelString}{' '}
+            <span className="vads-u-font-family--sans vads-u-font-weight--normal vads-u-font-size--base vads-u-color--secondary-dark">
```

Comment:
jotting notes from our discussion
- We don't want to use a custom required here, but it is needed because otherwise it would appear after the follow on paragraph since they are both included in the ui title. (Which would be fixed by the issues noted above)